frontend/src/util/tinymceInit.js

```diff
@@ -130,15 +130,15 @@ import { htmlCodeToDocumentFragment } from 'tracim_frontend_lib'
 
         const getPosition = (e) => {
           const toolbarPosition = $($editor.getContainer()).find('.mce-toolbar-grp').first()
-          const selectionNode = $($editor.selection.getNode())
-          const nodePosition = selectionNode.position()
-          const nodeOffset = selectionNode.offset()
-          const nodeHeight = selectionNode.height()
+          const nodePosition = $editor.selection.getNode().getBoundingClientRect()
           const isFullscreen = $editor.getContainer().className.includes('mce-fullscreen')
+
+          const topPosition = (isFullscreen ? $editor.getContainer().offsetTop : 0) + nodePosition.top + toolbarPosition.height()
+
           return {
-            top: (isFullscreen ? $editor.getContainer().offsetTop + nodePosition.top : nodePosition.top) + toolbarPosition.height(),
-            isSelectionToTheTop: nodePosition.top === 0,
-            selectionHeight: (nodeOffset.top * 2) + nodeHeight,
+            top: topPosition,
+            bottom: (isFullscreen ? $editor.getContainer().offsetTop : 0) + nodePosition.bottom + toolbarPosition.height(),
+            isSelectionToTheTop: topPosition < 280, // INFO - GB - 2021-07-06 - 280px is the autocomplete height
```

Comment:
```suggestion
            isSelectionToTheTop: topPosition < AUTOCOMPLETE_HEIGHT,
```

I suggest replacing the comment by a constant with a descriptive name :-)

Do you know where it is computed? Maybe there is a way to share this value in the two places?

Reply:
The real value is "auto", it depends on the number of elements. I tested with the number max and I got the value. Not ideal, but better than the last one that didn't works. There is an issue to better handle this autocompletion: https://github.com/tracim/tracim/issues/3544 I'm gonna add a point about it.

Reply:
I'm gonna add the constant too

Reply:
good :+1: